harness/tests/experiment/fixtures/pytorch_xor_model.py

```diff
@@ -312,6 +314,9 @@ def on_validation_step_start(self) -> None:
     def on_validation_step_end(self, metrics: Dict[str, Any]) -> None:
         self.validation_steps_ended += 1
 
+    def on_checkpoint_end(self, checkpoint_dir: pathlib.Path):
+        self.checkpoints_ended += 1
```

Comment:
Why do we need this value?

Reply:
To make sure this callback is being called once every checkpoint.